sympy/physics/vector/point.py

```diff
@@ -32,7 +32,8 @@ class Point:
     >>> O.acc(N)
     u1'*N.x + u2'*N.y + u3'*N.z
 
-    symbols() can be used to create multiple Points in a single step, for example:
+    ``symbols()`` can be used to create multiple Points in a single step, for
```

Comment:
Same here. Possible also for Point, but maybe that looks a bit weird as the full path must be included (I think?).